Packs/EclecticIQ/Integrations/EclecticIQIntelligenceCenterv3/EclecticIQIntelligenceCenterv3.py

```diff
@@ -2043,6 +2044,7 @@ def get_entity_by_id(eiq_api):
             outputs_prefix="EclecticIQ.EntityById",
             outputs=query_result,
             outputs_key_field="entity_title")
+    return None
```

Comment:
Same goes here, lets return a readable output message (rather than None) in case the query result does not satisfy any of the conditions.

Reply:
but it will change the flow of our existing customer